mmdet/models/utils/brick_wrappers.py

```diff
@@ -0,0 +1,50 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn.bricks.wrappers import NewEmptyTensorOp, obsolete_torch_version
+
+if torch.__version__ == 'parrots':
+    TORCH_VERSION = torch.__version__
+else:
+    # torch.__version__ could be 1.3.1+cu92, we only need the first two
+    # for comparison
+    TORCH_VERSION = tuple(int(x) for x in torch.__version__.split('.')[:2])
```

Comment:
You can use 
https://github.com/open-mmlab/mmdetection/blob/b1f97c1b6d2b3c735d8c5690952264c598c2f206/mmdet/__init__.py#L7

Reply:
We just follow https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/bricks/wrappers.py#L16.  If digit_version is adopted, I need special treatment for parrots.